robot/EDA/tests/browser/affiliations/affiliations_dropdowns_checkboxes.robot

```diff
@@ -0,0 +1,199 @@
+*** Settings ***
+Documentation
+...     Test Affiliations Settings:  
+...     All checkboxes disabled/enabled and all dropdowns 
+...     set to --None-- or one of the available options
+...     This tests the edit and save of the Affiliations 
+...     Settings checkboxes and dropdowns
+
+Resource        cumulusci/robotframework/Salesforce.robot
+Resource        robot/EDA/resources/EDA.robot
+Library         robot/EDA/resources/EDA.py
+Library         DateTime
+Library         cumulusci.robotframework.PageObjects
+...             robot/EDA/resources/AffiliationsPageObject.py
+
+Test Setup      Initialize test setup
+
+Suite Setup     Run keywords
+...             Initialize test data
+...             Open test browser
+
+Suite Teardown  Run keywords
+...             Close Browser 
+
+*** Test Cases ***
+
+Affiliations settings verify standard values
+    [tags]                                      unstable
+
+    Current page should be                      Listing                    
+    ...                                         hed__HEDA_Settings
```

Comment:
Does this need to be on a separate line or can i be on the same line as the Current page should be keyword?

Reply:
It can be on the same line.